doc/api/child_process.markdown

```diff
@@ -432,10 +194,21 @@ the existing process and uses a shell to execute the command.*
   * `stderr` {Buffer}
 * Return: ChildProcess object
 
-This is similar to [`child_process.exec()`][] except it does not execute a
-subshell but rather the specified file directly. This makes it slightly
-leaner than [`child_process.exec()`][]. It has the same options.
+The `child_process.execFile()` method is similar to [`child_process.exec()`][]
+except that it does not first spawn a shell. Rather, the specified `command` is
+spawned directly as a new process making it slightly more efficient than
+[`child_process.exec()`][]. The same options are support by both
+`child_process.exec()` and `child_process.execFile()`.
 
+    const exec = require('child_process').execFile;
```

Comment:
Sadly, this code example doesn't work even on Unix-like platforms. Since there's no shell wrapper,

1. `file` must be an absolute path to executable.
2. `args` must contain all the command line arguments for `file`.
3. All paths within `args` must be absolute as well.
4. Stdio redirection is not possible.

Here's an alternative I whipped up that should work on all Unix-like platforms. As always, feel free to use none, some, or all of it.

```js
const execFile = require('child_process').execFile;
const child = execFile('/bin/cat', ['/etc/paths'], (error, stdout, stderr) => {
  if (error) {
    throw error;
  }
  console.log(stdout);
});
```